process_testcase.go

```diff
@@ -302,24 +302,31 @@ func (v *Venom) runTestSteps(ctx context.Context, tc *TestCase, tsIn *TestStepRe
 			printStepName := v.Verbose >= 1 && !fromUserExecutor
 			v.setTestStepName(tsResult, e, step, &ranged, &rangedData, rangedIndex, printStepName)
 
```

Comment:
Diff below looks complicated but it actually just move all the code related to `v.RunTestStep` to the `else` block of the `if skip` conditional